xblocks_contrib/poll/inheritance.py

```diff
@@ -0,0 +1,49 @@
+"""
+Support for inheritance of fields down an XBlock hierarchy.
+"""
+
+
+from xblock.runtime import KeyValueStore
+
+
+class InheritanceKeyValueStore(KeyValueStore):
```

Comment:
Does this need to be in this repo? It looks like its usage is commented out in poll.py

Reply:
No, it is not needed, i tried something and then forgot to remove it. It has removed now.